### PR TITLE
125-YS- Send shortlist to shortlist table 

### LINKED ISSUE #125: TA- Send shortlist to shortlist table
- Create shortlist model
     - mentor_id
     - cohort_id
     - mentee_id
     - rank
- Send shortlist to database

### PR DESCRIPTION
**Pull request by YoungSong99**
Initial matching algorithm is done.

You can test it after create short list as mentee and 
go to http://0.0.0.0:3000/matches/new and clicked the create match.

We may clean up the code a lot for example creating shortlist controller etc.
Hope this help to start. and let me know if there is any questions. ;)

closes #125 
starts issue 126

### CODE REVIEW

**Review by MayaS1111 — APPROVED** [review]
Thank you so much for starting the matching process without being asked. I will help me a lot and it saves me a lot of time!

File: app/models/short_list.rb
Comment (brvarner) [inline app/models/short_list.rb:1]: Love this new model!

File: config/routes.rb
Comment (brvarner) [inline config/routes.rb:1]: I love the nested route here

**Review by brvarner — APPROVED** [review]
Great work as usual! I love the concise syntax you created for all of this.

**Review by rezahusain — APPROVED** [review]
Nice clean code Young! Approved.

**Review by borvux — APPROVED** [review]
Looks awesome!

**Pull request closed without merging**